math/minuit2/src/VariableMetricBuilder.cxx

```diff
@@ -69,15 +69,21 @@ FunctionMinimum VariableMetricBuilder::Minimum(const MnFcn &fcn, const GradientC
    // at exit of this function the BuilderPrintLevelConf object is destructed and automatically the
    // previous level will be restored
 
-   if (seed.Parameters().Vec().size() == 0) {
-      return FunctionMinimum(seed, fcn.Up());
-   }
-
    //   double edm = Estimator().Estimate(seed.Gradient(), seed.Error());
    double edm = seed.State().Edm();
 
    FunctionMinimum min(seed, fcn.Up());
 
+   if (seed.Parameters().Vec().size() == 0) {
+      print.Error("No free parameters.");
```

Comment:
Now this adds an Error message in stressRooStats, where in some case the conditional fit are made and likelihood has no free parameters. 
I think we could just add a Warning message here.

Reply:
Ok, I changed.